### PR TITLE
Improve Scapy's installation doc

### PR DESCRIPTION
**Pull request by gpotter2**
This PR:
- cleans up Scapy's installation doc:
    - Removes OpenBSD-specific dependencies page: it is outdated, since they can all be installed with pip nowadays.
    - Merges very similar platform-specific installation pages
    - Fixes some grammar odities (e.g. "Optional software for special features" -> "Optional dependencies")
    - Add documentation about Scapy's bundles (`basic`, `complete`), available since 2.4.3. (for now they are specified with `--pre`, but that will be updated once 2.4.3 comes out)
    - Fix missing quotes & a few weird spacing that would slightly mess up the indentation
- Since Pip > 9, it will automatically handle the versioning of dependencies. Therefore, as advised by https://github.com/pypa/setuptools/pull/1743#issuecomment-482994157, let Pip handle this.
- mark Scapy as not `zip_safe`. This slightly speeds up the install, as pip/setuptools don't have to guess it
- makes the Code Quality test run on Python 3.6, to remove the Python 2.7 deprecation warnings

### CODE REVIEW

File: doc/scapy/installation.rst [hunk 1]
```diff
@@ -92,80 +97,82 @@ Then you can always update to the latest version::
...
+    >>> p=readpcap("myfile.pcap")
+    >>> p.conversations(type="jpg", target="> test.jpg")
+
+  .. note::
+    ``Graphviz`` and ``ImageMagick`` need to be installed separately, using your platform-specific package manager.
+
 * 3D graphics. ``trace3D()`` needs `VPython-Jupyter <https://github.com/BruceSherwood/vpython-jupyter/>`_.
 
-    Jupyter-IPython is installable via `pip install vpython`
+  Jupyter-IPython is installable via ``pip install vpython``
```
Comment (guedou) [inline doc/scapy/installation.rst]: s/Jupyter-IPython/VPython-Jupyter/ ?

**Comment by guedou** [comment]
Cool PR. This my small comment.

**Comment by guedou** [comment]
Do you know how this will work with twine when we will release a new version?

**Comment by gpotter2** [comment]
If it worked previously (that was already packed on the 2.4.3rc), It should work as well.

It removes complexity, so it shouldn't work less well :smile:

**Pull request merged**